helios-services/src/test/java/com/spotify/helios/master/resources/HostsResourceTest.java

```diff
@@ -80,6 +81,7 @@ public void setUp() {
           .build();
 
       when(model.getHostStatus(host)).thenReturn(hostStatus);
```

Comment:
getHostStatus(host) should not be called anymore, so do you still need the `when()`? removing this catches the case where it is called unexpectedly